src/detect-engine-threshold.c

```diff
@@ -494,7 +494,7 @@ static int ThresholdHandlePacketSuppress(Packet *p,
     return ret;
 }
 
-static inline void RateFilterSetAction(Packet *p, PacketAlert *pa, uint8_t new_action)
+static inline void RateFilterSetAction(PacketAlert *pa, uint8_t new_action)
```

Comment:
Nice, I wonder if compiler can help us find more, without flagging the generic callbacks

Reply:
Not understanding the comment

Reply:
Wunused-argument yields too many warnings because there are certain callbacks with a precise prototype where every implementation does not use all arguments

Reply:
Cf https://github.com/OISF/suricata/pull/11080